### ISSUE TITLE
Need more info on `failed to receive server preface within timeout`

### ISSUE THREAD

**Issue by aprasolova** [nspcc-dev/neofs-node#1341]
## Is your feature request related to a problem? Please describe.
I try to create a container in neofs-dev-env via neofs-cli and get an error
```
rpc error: write request: rpc error: code = Unavailable desc = failed to receive server preface within timeout
```
When I read the logs I see no error messages in IR, SN and morph node. I'd like to get more information on the error cause to be able to solve it.

## Describe the solution you'd like
I'd like to see the error forwarded through all the engaged component with additional information added on every step. If it's techically possible, I'd like to get a message like "I aborted the request due to timeout, because an X component is delayed to persist some record" from CLI.

## Additional context
NeoFS: 0.28.0-rc.3
NeoGO: v0.98.2
dev-env: 19a52fd0575d1af4a4c87ceaccca2168cf71b66a

**Comment by cthulhu-rider** [comment]
`code = Unavailable desc = failed to receive server preface within timeout` part of an error is thrown by `grpc` lib. We can extend error trace with intermediate steps between it and `write request`.

**Comment by cthulhu-rider** [comment]
@aprasolova the problem should be fixed in `>=v0.28.1`
